translations/README.md

```diff
@@ -0,0 +1,23 @@
+# Translation files
+
+A few notes related to the translation of hardcoded strings in themes and modules.
+
+## Importing translations
+
+Visit `/admin/config/regional/translate/import` and upload the file of your choice. Select all three configuration options when importing the file.
+
+## Exporting translations
+
+Visit `/admin/config/regional/translate/export` and download the file of your choice. Select all three configuration options when exporting your file.
+
+## Creating translations
+
+As an admin, visit `/admin/config/regional/translate`
+
+When using the translation interface, you must load a page containing the string to be translated in the NON-original language at least once. So for instance if there's a new paragraph type to be translated, the process would be:
+
+1. As admin, add the new paragraph type to an English node. This is because for the GHO website, you cannot add new paragraphs to translations of pages. It must first be in the original.
```

Comment:
GHO-specific notes. They might still be relevant site sites using Paragraphs to build content, but we can scrub the GHO reference.